ale/drivers/base.py

```diff
@@ -195,13 +264,51 @@ def detector_center(self):
     def spacecraft_name(self):
         return self._label['MISSION_NAME']
 
+    @property
+    def starting_detector_line(self):
+        return 1
+
+    @property
+    def starting_detector_sample(self):
+        return 1
+
+    @property
+    def detector_sample_summing(self):
+        return 1
+
+    @property
+    def detector_line_summing(self):
+        return self._label.get('SAMPLING_FACTOR', 1)
+
+    @property
+    def filter_number(self):
+        return self._label.get('FILTER_NUMBER', 0)
```

Comment:
Filter Number is not something used by every instrument. Generally, filters are used by framing camera instruments.